### PR TITLE
RDoc-2013 Updated and edited Getting Started and Setup Wizard articles

### PR DESCRIPTION
**Pull request by serezhub**

### CODE REVIEW

File: Documentation/5.1/Raven.Documentation.Pages/start/getting-started.markdown [hunk 1]
```diff
@@ -229,9 +254,9 @@ RavenDB is a Document Database. All stored objects are called `documents`. Each
 
 ### Example II - Loading
 
-The `Session` was designed to help the user write efficient code easily. For example, when a document is being loaded (`.Load`) from the server, there is an option to retrieve additional documents in the same request (using `.Include`), keeping the number of expensive calls to minimum.
+The `Session` was designed to help the user write efficient code easily. For example, when a document is being loaded (`.Load`) from the server, there is an option [to retrieve additional documents in the same request](../../client-api/session/loading-entities#load-with-includes) (using `.Include`), minimizing the number of expensive calls.
```
Comment (reebhub) [inline Documentation/5.1/Raven.Documentation.Pages/start/getting-started.markdown:257]: the link in "to retrieve additional documents in the same request" doesn't work.
please verify all the links in this article to make sure we didn't mess up anything.

Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/start/getting-started.markdown]: I fixed it and checked all of the links in the article.

Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/start/getting-started.markdown]: Then committed again.

Comment (reebhub) [reply Documentation/5.1/Raven.Documentation.Pages/start/getting-started.markdown]: seems like the link is now fixed for 5.2 and 5.3 and completely removed from all versions under 5.2

File: Documentation/5.1/Raven.Documentation.Pages/start/getting-started.markdown [hunk 2]
```diff
@@ -144,9 +168,10 @@ Our GUI, the RavenDB Management Studio, comes **free** with **every license type
 
 {SAFE/}
 
-After installation and setup, the Studio can be accessed via the browser using the `ServerUrl` or the `ServerPublicUrl` value e.g. `http://localhost:8080`.
+After installation and setup, the Studio can be accessed via the browser using the `ServerUrl` or the `ServerPublicUrl` value e.g. `http://localhost:8080`.  
+Whenever you run the server folder script `run.ps1` the Studio opens automatically in your browser.  
 
----
+--
```
Comment (reebhub) [inline Documentation/5.1/Raven.Documentation.Pages/start/getting-started.markdown:174]: not sure why you changed "---" to "--", in its current form it doesn't draw a line. please change it back.

Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/start/getting-started.markdown:174]: That wasn't intentional - I changed it.

**Review by reebhub — APPROVED** [review]

**Pull request merged**